helpful.el

```diff
@@ -46,7 +46,7 @@
 (require 'dash)
 (require 's)
 (require 'find-func)
-(require 'nadvice)
+(require 'advice)
```

Comment:
Why did you change this line?

Reply:
I saw that the builds were (are?) red on master and checked travis to bind that it complained by nadvice. I didn't know that 'nadvice exists. I will fix it there